src/explore-education-statistics-frontend/src/modules/find-statistics/PublicationReleasePage.tsx

```diff
@@ -111,7 +116,13 @@ class PublicationReleasePage extends Component<Props> {
 
             <ReactMarkdown className="govuk-body" source={data.summary} />
 
-            <Details summary="Download data files">
+            <Details
+              summary="Download data files"
+              onToggle={(open: boolean) =>
+                open &&
+                logEvent('Downloads', 'Open download data files accordion')
```

Comment:
I'd suggest that adding an Event "label" which shows the current URL would be useful to this GA event (and all GA events that you're going to be adding).  Not something to block this PR but better to start this sooner rather than later.

Reply:
Yep i'd agree we should be capturing the current url